contracts/utils/ExtendedMulticall.sol

```diff
@@ -36,4 +36,24 @@ contract ExtendedMulticall is Multicall {
     function getBlockBasefee() external view returns (uint256) {
         return block.basefee;
     }
+
+    /// @notice Receives and executes a batch of function calls on this contract
+    /// @dev Unlike Openzeppelin.multicall(), this function does not revert when a call fails
+    /// @param data Array with each function call data
+    /// @return succeeded Array with each call success condition
+    /// @return returnData Array with each call result
+    function tryMulticall(bytes[] calldata data)
+        external
+        returns (bool[] memory succeeded, bytes[] memory returnData)
```

Comment:
I think the names should be `success` (uncountable) and `returndata` (already plural) to be consistent with https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Address.sol#L135 because this is extending the OZ implementation